src/vs/vscode.d.ts

```diff
@@ -9,10 +9,28 @@
 
 declare namespace vscode {
 
-	/**
-	 * The version of the editor.
-	 */
-	export var version: string;
+	export const env: {
+		/**
+		 * The version of the editor.
+		 */
+		version: string;
+
+		/**
+		 * locale, like de-ch, en-us
+		 */
+		locale: string;
```

Comment:
@dbaeumer We are looking into adding some vscode environment information. What do thing is needed for localisation support. Is a `locale` string enough?